vtexdocs/devportal
# Refactor/contexts

#### What is the purpose of this pull request?

To improve code readability and maintaining by refactoring the contexts of the application.

#### What problem is this solving?

Some variable names and types used in the sidebar and search contexts didn't reflect their meanings, making it difficult to understand the code in some circumstances.

#### How should this be manually tested?

Go to [this page](https://deploy-preview-51--elated-hoover-5c29bf.netlify.app/) and check if the the sidebar and search still work.

#### Screenshots or example usage

#### Types of changes
